generate.py

```diff
@@ -0,0 +1,58 @@
+import toml
+import sys
+import os
+
+methods = {
+   "apt": "$APT_GET",
+   "yum": "$YUM",
+   "dnf": "$DNF",
+   "apk": "$APK",
+   "pacman": "$PACMAN"
+}
+
+def get_method_case(method):
+   return "[ ! -z "+methods[method]+"_CMD ]; then\n"
```

Comment:
What will happen if the given `method` is not available in the `methods` dictionary? Handle these error flow and add proper logs